### PR TITLE
Update BIM development setup script to use .NET 6 and update xeokit-metadata

### PR DESCRIPTION
**Pull request by wielinde**
https://community.openproject.org/projects/openproject/work_packages/44335

### CODE REVIEW

File: modules/bim/bin/setup_dev.sh [hunk 1]
```diff
@@ -12,11 +12,9 @@ if [[ $EUID -ne 0 ]]; then
    exit 1
 fi
 
-# Specifics for BIM edition
-wget -qO- https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
-wget -q https://packages.microsoft.com/config/debian/9/prod.list -O /etc/apt/sources.list.d/microsoft-prod.list
+# Specifics for BIM edition (Ubuntu)
 apt-get update -qq
-apt-get install -y dotnet-runtime-3.1
+apt-get install -y dotnet-runtime-6.0
```
Comment (crohr) [inline modules/bim/bin/setup_dev.sh]: @wielinde this is only available under ubuntu 22.04 / debian 12. So this dev script should probably check for that as well.

~~This also means that this is probably going to be an issue for packaging. I'll have a look at whether there are still Microsoft APT/YUM repos available for older distributions.~~ => it's fine

Comment (crohr) [inline modules/bim/bin/setup_dev.sh]:
```suggestion
apt-get install -y dotnet-runtime-6.0 wget unzip
```

**Comment by oliverguenther** [comment]
:warning: please rebase onto `release/12.3` and change the target branch

File: modules/bim/bin/setup_dev.sh (hunk 1 above)
Comment (wielinde) [reply modules/bim/bin/setup_dev.sh]: This script is just for developers. They can have so many different distros. Some have Mac. I am fine with not making it 100% fail proof. Especially as there are instructions printed at the end of the script that make manual intervention necessary anyways.

**Pull request merged**